azavea/raster-foundry
# Move and rename app server to api

## Overview

This PR renames the `app` subproject to `api` and moves it to its own package `com.azavea.rf.api`. It also renames the docker container from `app-server` to `api-server`.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations`

## Testing Instructions

 * `./scripts/test` should pass
 * Run the server `./scripts/server` and ensure the app works as expected
 * Check the various scripts to make sure they work (`./scripts/console` etc)

Connects #677
